docs/source/download.rst

```diff
@@ -1,111 +1,200 @@
-=============
-Obtain Devito
-=============
+===========================
+Download and Install Devito
+===========================
+
+There are two main approaches to installing Devito.
+
+- `Docker installation`_, for those looking for a quick “out-of-the-box” way to try Devito
+- `pip/conda installation`_, for those looking to use Devito as part of a project alongside other packages or/and looking to develop with Devito
 
-There are three main approaches to installing Devito. For those looking for the
-least-friction way to try Devito, we recommend the Docker route. For those
-looking to use Devito alongside other packages as part of a project, we support
-pip installation. If you're looking to develop for Devito, you might benefit from
-using the included conda environment that includes all the bells and whistles we
-recommend when developing for Devito.
 
 Docker installation
 -------------------
+
```

Comment:
IMHO we should drop this entire section and just link to the README on github

if there's something here that is not in the README on github, we should add it there

Reply:
@mloubout

Reply:
These are the the `docker-compose` instruction. Current changes don't make sense to me the instructions should stay the same given the `docker-compose.yml` is setup correctly.

I'm still not sure why we have those as recommended instructions since None of the main devs use it and most people end up using conda or pip to use Devito in combination with something else.

Reply:
> Current changes don't make sense to me
Which changes do you mean?